### PR TITLE
docs and changelog for `nomad config validate`

### PR DESCRIPTION
**Pull request by tgross**
For https://github.com/hashicorp/nomad/pull/9198

### CODE REVIEW

File: .changelog/9198.txt [hunk 1]
```diff
@@ -0,0 +1,33 @@
+```release-note:improvement
+cli: add Usage: nomad [-version] [-help] [-autocomplete-(un)install] <command> [args]
```
Comment (jrasell) [inline .changelog/9198.txt:2]: are we expecting the entire Nomad usage here?

File: website/content/docs/commands/config/index.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+## Usage
+
+Usage: `nomad config <subcommand> [options]`
+
+Run `nomad config <subcommand> -h` for help on that subcommand. The
+following subcommands are available:
+
+- [`config validate`][validate] - Validate configuration files
+
+[gc]: /docs/commands/config/validate 'Validate configuration files'
```
Comment (jrasell) [inline website/content/docs/commands/config/index.mdx:21]:
```suggestion
[validate]: /docs/commands/config/validate 'Validate configuration files'
```

**Review by jrasell — CHANGES_REQUESTED** [review]
The changelog entry seems incorrect and a link tag (or whatever it's proper name is) needs to be update, other than that looks good.

File: .changelog/9198.txt (hunk 1 above)
Comment (tgross) [reply .changelog/9198.txt:2]: LOL, I did this at my command line:

```
 new-cl improvement 9198 "cli: add `nomad config validate` command to check configuration files without an agent"
```

And then didn't look at the resulting file. 😊

**Review by jrasell — APPROVED** [review]
LGTM!

**Pull request merged**